### PR TITLE
ARKit Updates

### PR DESCRIPTION
**Pull request by EvanBacon**
Added the ability to:
* Get raw feature points
* Get estimated light
* Enable / disable light estimation
* Enable / disable plane detection

**These features are not yet implemented in Expo**

### CODE REVIEW

**Comment by pierotofy** [comment]
@EvanBacon wondering if you have published your changes to the expo client/lib somewhere?

**Comment by EvanBacon** [comment]
@pierotofy these changes should be in SDK v24! (Currently we are on v23)

**Comment by mattg73** [comment]
@EvanBacon any plans for ARCore support in the SDK?

**Comment by EvanBacon** [comment]
@mattg73 investigating integration now ❤️😁

File: ExpoTHREE.js [hunk 1]
```diff
@@ -26,9 +24,9 @@ export const createRenderer = ({ gl, ...extra }) =>
       style: {},
       addEventListener: () => {},
       removeEventListener: () => {},
-      clientHeight: gl.drawingBufferHeight,
```
Comment (nikki93) [inline ExpoTHREE.js]: Let's leave commas in at the ends of lines.

File: ExpoTHREE.js [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 // THREE.js tries to add some event listeners to the window, for now
 // just ignore them.
 
 if (!window.addEventListener) {
   window.addEventListener = () => {};
 }
 
-
-const THREE = require('three');
+const THREE = require("three");
```
Comment (nikki93) [inline ExpoTHREE.js]: Let's stick to single-quoted strings.

File: ExpoTHREE.js [hunk 3]
```diff
@@ -87,3 +85,55 @@ export const createARBackgroundTexture = (arSession, renderer) => {
...
+};
+
+/**
+ Enable or disable light estimation.
+ @discussion Enabled by default.
+
+ isLightEstimationEnabled
+ */
+export const enableLightEstimation = (arSession, enable) => {
+  return NativeModules.ExponentGLViewManager.enableLightEstimation(
```
Comment (nikki93) [inline ExpoTHREE.js]: Needs to reflect updated API (`setIs.*Enabled` as opposed to `enable.*`).

**Pull request merged**